spec/features/early_career_payments_claim_spec.rb

```diff
@@ -0,0 +1,76 @@
+require "rails_helper"
+
+RSpec.feature "Teacher Early Career Payments claims" do
+  scenario "Teacher makes claim for 'Early Career Payments' claim" do
+    visit new_claim_path(EarlyCareerPayments.routing_name)
+    expect(page).to have_link(href: EarlyCareerPayments.feedback_url)
+
+    # TODO [PAGE 00] - Landing (start)
```

Comment:
Again, shouldn't be adding TODOs. Not sure why this needs to be written out and merged into master when we already have requirements elsewhere, adding here is less agile. Should be fairly obvious where things need to be added.

Reply:
@nomasprime I explained in Standup this morning that I had added this as sign posts for our Investment colleagues so they could easily find where to add the parts for their stories. I also explicitly asked that you not raise this in the PR as I had explained the logic.

While I don't want to deliver simply for delivery sake and to be a 'Yes' man, I want to be helping our Investment colleagues learn and deliver on the tight timescales we are up against